modules/appeals_api/app/models/concerns/appeals_api/appeal_scopes.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module AppealScopes
+    extend ActiveSupport::Concern
+
+    included do
+      scope :without_status_updates_since, lambda { |time|
+        status_update_table = AppealsApi::StatusUpdate.table_name
+        join_clause = <<~JOIN
+          LEFT JOIN #{status_update_table}
+          ON #{table_name}.id = CAST(#{status_update_table}.statusable_id as uuid)
+          AND #{status_update_table}.id IS NULL
+          AND #{status_update_table}.statusable_type = '#{sanitize_sql(klass.name)}'
+        JOIN
+        where("#{table_name}.updated_at <= ?", time)
```

Comment:
Note that here, where I'm selecting records `without_status_updates_since(time)`, I'm also checking that the records themselves haven't been updated since `time`. The acceptance criteria didn't mention this explicitly, but I think it makes sense. For example, if somehow a record reached 45 days old without ever having been moved to a different status (so it has no StatusUpdate records) but somehow that same record has been updated in a different way since then, its PII would not (yet) be removed, even though it was old enough.

Reply:
That seems fair. I know that we bounced between looking at the status updates or just `updated_at` on the appeal, so I think it's good to consider both.